OrcanodeMonitor/OrcanodeMonitor.csproj.user

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
revert adding this file. 
.user" files should not be checked in.
(I will add that suffix to .gitignore.)